<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGraph</class>
 <widget class="QWidget" name="QtGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGraph</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbBuild">
       <property name="text">
        <string>Build</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFunc">
       <property name="text">
        <string>F(x)=</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edFunc"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabLeft">
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tbConf">
        <attribute name="title">
         <string>Настройки</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Границы X</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QSlider" name="sliderXLimits">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksBelow</enum>
              </property>
              <property name="tickInterval">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="labelXFrom">
                <property name="text">
                 <string>От</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="edXFrom">
                <property name="inputMethodHints">
                 <set>Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
                </property>
                <property name="text">
                 <string>-10</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelXTo">
                <property name="text">
                 <string>До</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="edXTo">
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tbLog">
        <attribute name="title">
         <string>Log</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="teLog"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>edFunc</sender>
   <signal>returnPressed()</signal>
   <receiver>pbBuild</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
